lib/aws/google/version.rb

```diff
@@ -1,5 +1,5 @@
 module Aws
   class Google
-    VERSION = '0.1.8'.freeze
+    VERSION = '0.2.0'.freeze
```

Comment:
I think a dependency change like this warrants a minor version bump rather than just a patch, but I don't know exactly what standards we're trying to enforce here. Lemme know if anyone thinks otherwise!

Reply:
Agreed it's somewhere in between. Definitely not a major version bump.